app/services/chat_service.rb

```diff
@@ -38,8 +38,8 @@ def self.configure_url(chat)
                                                                     host: Rails.configuration.web_app_hostname)
   end
 
-  def send_update(chat, msg)
-    @telegram_api.sendMessage(chat_id: chat.telegram_chat_id, text: msg)
+  def send_update(chat, msg, disable_web_preview = false)
```

Comment:
Let's make preview disabled by default, I don't see where it's needed now. So, I would rename the var to 'web_preview': false